### DISCUSSION TITLE
Request - asterisks

### DISCUSSION THREAD

**Discussion by TheKnightWho** [psb1558/Junicode-font#124] [Ideas]
Hi - would it please be possible to include some additional asterisk code points?

I was thinking the asterisks in the [Dingbats](https://www.unicode.org/charts/PDF/U2700.pdf) and [Geometric Shapes Extended](https://www.unicode.org/charts/PDF/U1F780.pdf), but I appreciate that there are quite a lot!

**Comment by psb1558** [comment]
Maybe someday . . . when I've got time on my hands after finishing what's on the to-do list now.
